.github/workflows/test.yml

```diff
@@ -38,3 +38,6 @@ jobs:
           echo 'All of the above failures are maybe false errors.'
           echo 'See: https://github.com/RDFLib/sparqlwrapper/issues/192'
         fi
+    - name: Mypy
```

Comment:
Given how slow the tests are to run, it may be best to run mypy before the tests.

Reply:
fb1949f